service_implementation/advanced_plm.md

```diff
@@ -1 +1,456 @@
-# Beyond just files: Using PLM for versioning and metadata
\ No newline at end of file
+# Beyond just files: Using PLM for versioning and metadata
+This document describes how to use PLM services from _inside_ a service, i.e., by calling the PLM API. The text includes  example code.
+There are two types of PLM services for use in CAxMan; both are described below:
+1) those that are integrated in the GSS framework;
+2) the PLM Server native web-services.
+
+## PLM API through GSS
+
+The PLM Server has its own implementation of the GSS services, that is, PLM service endpoints can be obtained through GSS. GSS offers SOAP and REST services:

```

Comment:
Suggested change:
The PLM server has features which go beyond what is offered by the GSS service. In order to use these additional features, a PLM service endpoint needs to be obtained (use the `getDirectionInteractionEndpoint()` method of GSS):

Reply:
done